src/scss/_statements.scss

```diff
@@ -24,184 +24,176 @@
-  @include declaration('text-align', $drops--text-align);
-  @include declaration('text-decoration', $drops--text-decoration);
-  @include declaration('border-width', $drops--border-width);
-  @include declaration('border-style', $drops--border-style);
-  @include declaration('border-color', $drops--border-color);
-  @include declaration('border-radius', $drops--border-radius);
-  @include declaration('color', $drops--color);
-  @include declaration('box-shadow', $drops--box-shadow);
-  @include declaration('background', $drops--background);
-  @include declaration('background-color', $drops--background-color);
-  @include declaration('min-width', $drops--min-width);
-  @include declaration('max-width', $drops--max-width);
-  @include declaration('width', $drops--width);
-  @include declaration('z-index', $drops--z-index);
-  @include declaration('-include-drops--gutter', $drops--margin);
-
-  @include block-generation($drop--selector);
-}
+$drop--selector: selector(new 'drop' with $drops--syntax);
+
+@include statement($drop--selector,(
+  (font-size, $drops--font-size),
+  (font-family, $drops--font-family),
+  (font-weight, $drops--font-weight),
+  (text-transform, $drops--text-transform),
+  (letter-spacing, $drops--letter-spacing),
+  (line-height, $drops--line-height),
+  (text-align, $drops--text-align),
+  (text-decoration, $drops--text-decoration),
+  (border-width, $drops--border-width),
+  (border-style, $drops--border-style),
+  (border-color, $drops--border-color),
+  (border-radius, $drops--border-radius),
+  (color, $drops--color),
+  (box-shadow, $drops--box-shadow),
+  (background, $drops--background),
+  (background-color, $drops--background-color),
+  (min-width, $drops--min-width),
+  (max-width, $drops--max-width),
+  (width, $drops--width),
+  (z-index, $drops--z-index),
+  ('-include-drops--gutter', $drops--margin),
+));
+
+
 
 /**
  * caret
  */
-$drop-before--selector: selector--set-pseudo-element($drop--selector, 'before');
+$drop-before--selector: selector(set $drop--selector pseudo-element 'before');
 
-@include selector($drop-before--selector) {
-  @include declaration('content', '');
-  @include declaration('position', absolute);
-  @include declaration('z-index', 1);
+@include statement($drop-before--selector,(
+  (content, ''),
+  (position, absolute),
+  (z-index, 1),
 
-  @include declaration('-include-drops-caret--base', $drops-caret--base);
-  @include declaration('-include-drops-caret--shape', $drops-caret--height);
-  @include declaration('-include-drops-caret--color', $drops--border-color);
-  @include declaration('-include-drops-caret--position', null);
-  @include declaration('-include-drops-caret--margin', $drops--border-width);
+  ('-include-drops-caret--base', $drops-caret--base),
+  ('-include-drops-caret--shape', $drops-caret--height),
+  ('-include-drops-caret--color', $drops--border-color),
+  ('-include-drops-caret--position', null),
+  ('-include-drops-caret--margin', $drops--border-width),
```

Comment:
the syntax for mixins has changed with v4. It should be written in the way below

```scss
('@include drops-caret--base', $drops-caret--base),
('@include drops-caret--shape', $drops-caret--height),
('@include drops-caret--color', $drops--border-color),
('@include drops-caret--position', null),
('@include drops-caret--margin', $drops--border-width),
```